test/e2e/metrics/gateway/kyma_input_test.go

```diff
@@ -50,6 +50,7 @@ func TestKymaInput(t *testing.T) {
 	assert.BackendReachable(t, backend)
```

Comment:
Please check if we need this check 	`assert.BackendReachable(t, backend)` into other tests as well ?